docker-compose.env.yaml.example

```diff
@@ -0,0 +1,12 @@
+DEBUG=1
+ALLOWED_HOSTS=*
+APPLY_MIGRATIONS=1
+LOAD_DATADUMP=1
+CREATE_SUPERUSER=1
+LOAD_GEODATA=0
+
+TUNNISTAMO_BASE_URL=http://tunnistamo-backend:8000
+SOCIAL_AUTH_TUNNISTAMO_KEY=http://tunnistamo-backend:8000/project
+SOCIAL_AUTH_TUNNISTAMO_SECRET=<ENTER TUNNISTAMO CLIENT SECRET HERE>
+SOCIAL_AUTH_TUNNISTAMO_OIDC_ENDPOINT=http://tunnistamo-backend:8000/openid
+PROFILE_API_URL=http://tunnistamo-backend:8000/v1
```

Comment:
well, should probably be either `<ENTER YOUR PROFILE API URL>` or something like `http://localhost:8080/v1`

Reply:
In this case the default is the name of the service that is created in the Tunnistamo docker-compose setup, that is why it is named that way :) So I would say that it is a sane default in this case.

Reply:
it just looks like tunnistamo's own API. in production, for example, tunnistamo is `https://api.hel.fi/sso` and profile is `https://api.hel.fi/profiles`. having `http://tunnistamo-backend:8000/v1` as placeholder does not give a very good idea of what value should actually be here

Reply:
This value is a dev template, it is not there to show what should be there in production. Documentation should take care of that IMHO.